Userland/Utilities/find.cpp

```diff
@@ -730,12 +752,11 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         StringView arg { raw_arg, strlen(raw_arg) };
         if (arg == "-L") {
             g_follow_symlinks = true;
-        } else if (!arg.starts_with('-')) {
-            paths.append(LexicalPath(arg));
-        } else {
-            // No special case, so add back the argument and try to parse a command.
+        } else if (arg.starts_with('-') || arg == "!"sv) {
             args.prepend(raw_arg);
             command = parse_all_commands(args);
```

Comment:
This does not seem to work correctly: now, a `!` inverts all expressions after it while I'd expect it to only invert the adjacent expression.

For example, see the difference between:
- `find . -type d -iname '*video*'`
- `find . -type f -iname '*video*'`
- `find . ! -type d -iname '*video*'`

Reply:
This was a misunderstanding on my part about how this feature is supposed to work. I've changed it so that it now behaves as you suggest. I also changed the example picture to use something close to your commands. 

Also tested with Linux `find`, which also behaves this way.

Side note: we don't seem to handle doing something like: `find . ! '(' -type d -iname 'video*' ')'` correctly due to the way parentheses are parsed at the moment. That bug isn't directly related to this PR though.